SDNExpress/scripts/FabricConfig.psd1

```diff
@@ -40,6 +40,19 @@
             #The HostUsername must contain the Domain name in the format DOMAIN\User
             HostUsername = '<< Replace >>'                                    #Example: CONTOSO\al
             HostPassword = '<< Replace >>'                                    #Example: MySuperS3cretP4ssword
+            
+            #iDNS IP address and resource ID. The creds are same as the domain creds
+            iDNSCredentialResourceId = "c6abefg6-44fb-45f6-89ec-5ebd890a144f"            
+            iDNSAddress= '<< Replace >>'                                      #Example: "10.0.0.7"
+            iDNSZoneName = '<< Replace >>'                                    #Example: "contoso.local"
+            DnsPort=53                                                        #Standard DNS port. DO NOT CHANGE            
+            DnsIPAddress="169.254.169.254"                                    #Fixed IP address understood by DNS proxy. DO NOT CHANGE
+            DnsProxyServiceName="DnsProxy"                                    #Name of the service. DO NOT CHANGE
+            iDNSMacAddress = '<< Replace >>'                                  #Example "AA-BB-CC-AA-BB-CC"
+
+            #iDNS Proxy Registry Hives
+            iDnsVfpPath = "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\NcHostAgent\Parameters\Plugins\Vnet\InfraServices\DnsProxyService"                                                                                          
```

Comment:
Same comment as FabricConfig.example.psd1 - lets move these fixed registry paths to the ps1 rather than exposing in the configuration file.